src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -1075,6 +1080,37 @@ private async Task<RepositoryBranchUpdate> GetRepositoryBranchUpdate()
 
         private static string GetNewBranchName(string targetBranch) => $"darc-{targetBranch}-{Guid.NewGuid()}";
 
+        private static async Task<List<(UpdateAssetsParameters update, List<DependencyUpdate> deps)>> ReplaceFromDependency(
+            IRemote darcRemote,
+            string targetRepository,
+            string targetBranch,
+            TargetRepoDependencyUpdate targetRepositoryUpdates
+            )
+        {
+            List<DependencyDetail> targetBranchDeps = (await darcRemote.GetDependenciesAsync(targetRepository, targetBranch)).ToList();
+
+            // 
```

Comment:
I guess the whole method could use a comment what it does and why we have it

Reply:
Good point, added